quantum/process_keycode/process_tap_dance.c

```diff
@@ -167,7 +166,11 @@ void matrix_scan_tap_dance() {
         if (action->custom_tapping_term > 0) {
             tap_user_defined = action->custom_tapping_term;
         } else {
+#ifdef TAPPING_TERM_PER_KEY
             tap_user_defined = get_tapping_term(action->state.keycode, NULL);
+#else
+            tap_user_defined = 200;
```

Comment:
Having a hard coded `200` here seems like a step backwards, is the following not valid here?

```suggestion
            tap_user_defined = TAPPING_TERM;
```

Reply:
You're correct, and I'm not sure why I went for that, TBH.